src/overlays/gamestates/ovl_debug_opening/z_debug_opening.c

```diff
@@ -0,0 +1,316 @@
+    GfxPrint_SetColor(printer, 2, 205, 250, 255);
+    GfxPrint_SetPos(printer, 20, 10);
+    GfxPrint_Printf(printer, gSaveContext.fileNum == 0xff ? "Debug Save" : "New Save");
+
+    for (i = 0; i < 5; i++) {
+        GfxPrint_SetColor(printer, 255, 255, 255, 255);
+        GfxPrint_SetPos(printer, 9, 10 + i);
+        if (i == this->currentOption) {
+            // highlight the currently selected option
+            GfxPrint_SetColor(printer, 237, 226, 7, 255);
+        }
+        GfxPrint_Printf(printer, sOptionInfo[i].name);
+    }
+}
+
+void DebugOpening_DrawBase(GameState* thisx) {
+    DebugOpeningState* this = (DebugOpeningState*)thisx;
+    GraphicsContext* gfxCtx = this->state.gfxCtx;
+    GfxPrint* printer;
+
+    OPEN_DISPS(gfxCtx, "../z_debug_opening.c", __LINE__);
+
+    gSPSegment(POLY_OPA_DISP++, 0x00, NULL);
+    Gfx_SetupFrame(gfxCtx, true, 0, 0, 0);
+    SET_FULLSCREEN_VIEWPORT(&this->view);
+    View_Apply(&this->view, VIEW_ALL);
+    Gfx_SetupDL_28Opa(gfxCtx);
+
+    printer = alloca(sizeof(GfxPrint));
+    GfxPrint_Init(printer);
+    GfxPrint_Open(printer, POLY_OPA_DISP);
+
+    GfxPrint_SetColor(printer, 220, 0, 17, 255);
+    GfxPrint_SetPos(printer, 12, 1);
+    GfxPrint_Printf(printer, "Zelda Debug Boot");
+
+    GfxPrint_SetColor(printer, 255, 255, 255, 255);
+    GfxPrint_SetPos(printer, 12, 2);
+    GfxPrint_Printf(printer, "Welcome ");
+
+    GfxPrint_SetColor(printer, 0, 160, 220, 255);
+    GfxPrint_Printf(printer, "%s!", gBuildAuthor);
+
+    GfxPrint_SetColor(printer, 255, 255, 255, 255);
+    GfxPrint_SetPos(printer, 0, 3);
+    GfxPrint_Printf(printer, this->seperatorString);
+
+    GfxPrint_SetPos(printer, 1, 23);
+    GfxPrint_Printf(printer, this->controlGuideString, this->page);
+
+    switch (this->page) {
+        case OPTIONS_PAGE:
+            DebugOpening_DrawOptions(thisx, printer);
+            break;
+        case COMMANDS_PAGE:
+            DebugOpening_DrawCommands(this, printer);
+            break;
+        case BUILDINFO_PAGE:
+            DebugOpening_DrawBuildInfo(this, printer);
+            break;
+        default:
+            break;
+    }
+
+    DebugOpening_DrawCommitInfo(this, printer);
+
+    POLY_OPA_DISP = GfxPrint_Close(printer);
+    GfxPrint_Destroy(printer);
+
+    CLOSE_DISPS(gfxCtx, "../z_debug_opening.c", __LINE__);
+}
+
+void DebugOpening_Main(GameState* thisx) {
+    DebugOpeningState* this = (DebugOpeningState*)thisx;
+
+    if (this->page == OPTIONS_PAGE) {
+        DebugOpening_ControlOptions(&this->state);
+    }
+    DebugOpening_DrawBase(thisx);
+    if (CHECK_BTN_ANY(this->state.input[0].press.button, BTN_R) && this->page != BUILDINFO_PAGE) {
+        this->page++;
+        Audio_PlaySfxGeneral(NA_SE_SY_FSEL_CURSOR, &gSfxDefaultPos, 4, &gSfxDefaultFreqAndVolScale,
+                             &gSfxDefaultFreqAndVolScale, &gSfxDefaultReverb);
+    } else if (CHECK_BTN_ANY(this->state.input[0].press.button, BTN_L) && this->page != OPTIONS_PAGE) {
```

Comment:
I think it should be using ``BTN_Z`` because if feels more natural than L with an n64 controller and it makes it less weird on a gamecube controller